stripe-ui-core/src/main/java/com/stripe/android/uicore/text/Html.kt

```diff
@@ -62,14 +75,16 @@ data class EmbeddableImage(
 fun Html(
     html: String,
     modifier: Modifier = Modifier,
-    imageGetter: Map<String, EmbeddableImage> = emptyMap(),
+    localImageLoader: Map<String, EmbeddableImage> = emptyMap(),
+    remoteImageLoader: StripeImageLoader = StripeImageLoader(LocalContext.current),
     color: Color = Color.Unspecified,
     style: TextStyle = LocalTextStyle.current,
     enabled: Boolean = true,
     urlSpanStyle: SpanStyle = SpanStyle(textDecoration = TextDecoration.Underline),
-    imageAlign: PlaceholderVerticalAlign = PlaceholderVerticalAlign.AboveBaseline
+    imageAlign: PlaceholderVerticalAlign = PlaceholderVerticalAlign.AboveBaseline,
+    onRemoteImagesLoaded: () -> Unit = {},
```

Comment:
What's the use case for this callback?

Reply:
This will be used for UME. We need a way to know when all the images are done downloading, since Html is responsible for downloading the images. When all the images are downloaded, then we can successfully return the fully rendered UME view back to the merchant.

If we don't use this, we will give back a half baked UME view and the user may see some jitteriness as the images finish downloading and finally get rendered into the view.

Reply:
Identity is [using](https://github.com/stripe/stripe-android/blob/ecba24292cb093456fb5ea943b709736082bbe11/identity/src/main/java/com/stripe/android/identity/ui/ConsentScreen.kt#L207) Html too, would this signature change break it?

Reply:
@ccen-stripe this change will allow you to render images by specifying drawables (by id), bitmaps. If you don't specify in the arguments, then the img source will be attempted to be rendered by `StripeImageLoader`. Looking at the code you provided, it seems like it shouldn't break anything.